### PR TITLE
Weno follow up

### PR DESCRIPTION
**Pull request by sjpadgett**
- correct interval for WenoExchange
- add after actions to enable, disable and delete background tasks.

Fixes #

### CODE REVIEW

**Comment by sjpadgett** [comment]
@dataengr More changes you may want to note.
@juggernautsei If interested in clean up pattern.
@bradymiller Unsure how you want to review but anything you note, say in merged original, I'll fix here. thanks!

File: interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php
Comment (juggernautsei) [inline interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php:1]: I see that you dropped this file in the root of the module. I have read the entire file. I understand most of what it is doing.  Is the pattern for the future of module building?

Comment (sjpadgett) [reply interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php:1]: only if needed to do specific module manager actions like remove a global when module is disabled etc.

File: interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 use OpenEMR\Modules\WenoModule\Services\WenoPharmaciesJson;
 
 if (isset($_GET['key']) && !empty(isset($_GET['key']))) {
+    // called from old v6.0.0 requirement.
```
[the remark this answers is not in the document]
Comment (sjpadgett) [reply interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php:15]: I don't think so and is fine

**Comment by sjpadgett** [comment]
Merging soon. If anyone has any further issue then by all means be my guess and change. @dataengr 
- Recommend you test for QA especially the pharmacy and prescription background tasks weno requirements for updates.
- If specific user doesn't have credentials then weno widget shouldn't show.
- Cleanup DB from v6.0.0 weno background tasks i.e remove. Actually I think i'll go ahead and do before merge so not forgotten.
- If any reporting requires encounter then follow up. I don't see any and Sherwin added as openemr requirement and not weno.

File: interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (stephenwaite) [reply interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php:15]: but anyone could hit this file with a get request key param and start weno?

**Comment by sjpadgett** [comment]
okay @dataengr Removed what I think is old left behinds for old weno feature and I see no usage by module but correct me if im wrong.

File: interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php [hunk 2]
```diff
@@ -11,16 +11,6 @@
...
-    // called from old v6.0.0 requirement.
-    start_weno();
-}
-function start_weno()
-{
-    $logsync = new LogProperties();
-    $logsync->logSync();
-}
-
 function downloadWenoPharmacy()
```
Comment (stephenwaite) [inline interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php:14]: southpark_cheering

**Comment by dataengr** [comment]
@sjpadgett, the method start_weno is no longer in use.

**Comment by sjpadgett** [comment]
it was a function and I removed it

**Comment by sjpadgett** [comment]
I think I did pretty good to make this module whole and clean. I'm sending back to @omegasystemsgroup to follow up with testing and any bug fixes I caused by making whole for openemr or otherwise missed in my testing which I wasn't charged with.
Merging and good luck Weno!

File: interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php (hunk 2 above)
Comment (bradymiller) [reply interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php:14]: fare thee well start_weno() :)

**Comment by bradymiller** [comment]
2DV-2

File: interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php
Comment (juggernautsei) [reply interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php:1]: So, for my first use of this feature, I will be adding it to the quest lab module. 
https://github.com/juggernautseinc/oe-quest-lab-hub
It has pieces that would need to be removed from background services and global.

Comment (juggernautsei) [reply interface/modules/custom_modules/oe-module-weno/ModuleManagerAfterActionListener.php:1]: How do I get a module code review? @bradymiller

**Pull request merged**